lps/simulare/src/components/FormRegister.tsx

```diff
@@ -133,6 +133,7 @@ const FormRegister = () => {
       moveSectionDown?.();
     }
   };
+  const iconColor = '#24BB78';
```

Comment:
@RayzaOliveira `color = iconColor;` is equal to `color = 'primary;', right? The problem using `iconColor = '#24BB78';` is that if we change our theme, this color won't be changed. We also need to use colors from theme.